lib/database/models/tag/route_tag.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:drift/drift.dart';
```

Comment:
Does this class act as a connector that associates routes to tags?

Reply:
Yeah basically instead of storing a list in the db we want to do a separate table.